src/components/LeftSidebar/Sponsors.astro

```diff
@@ -24,6 +24,19 @@
 				fill="currentColor"></path>
 		</svg>
 	</a>
+	<a
+		href="https://idx.dev/?utm_source=astro&utm_medium=astro&utm_campaign=astro"
+		aria-label="Google IDX"
```

Comment:
Oh, one thing I missed. The label here probably should be “Project IDX” to actually match the text in the logo, otherwise it would be a bit odd:
```suggestion
		aria-label="Project IDX"
```